hydroflow_lang/src/graph/ops/source_interval.rs

```diff
@@ -17,15 +17,16 @@ use super::{
 /// as possible, and the `Instant` will be the missed time, not the late time.
 ///
 /// Note that this requires the hydroflow instance be run within a [Tokio `Runtime`](https://docs.rs/tokio/1/tokio/runtime/struct.Runtime.html).
-/// The easiest way to do this is with a [`#[tokio::main]`](https://docs.rs/tokio/1/tokio/attr.main.html)
+/// The easiest way to do this is with a [`#[hydroflow_main]`](https://docs.rs/tokio/1/tokio/attr.main.html)
```

Comment:
Link is wrong, would be <https://hydro-project.github.io/hydroflow/doc/hydroflow/macro.hydroflow_main.html>